jfrog_metrics/datadog_checks/jfrog_metrics/__about__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```

Comment:
version in changelog and __about.py__ should be the same. Can you please change accordingly

Reply:
why do we have version 0.1.0 in changelog? should not it be 0.0.1

Reply:
Since it is the first major version, we're starting with 0.1.0